packages/widgets/src/components/LinksTabBar/LinksTabBar.tsx

```diff
@@ -0,0 +1,14 @@
+import { NavLink } from 'react-router-dom';
+
+import { LinksTabBarProps } from './types';
+import { LinksTabBarBase } from './LinksTabBarBase';
+
+export const LinksTabBar = ({ as, className, links }: LinksTabBarProps) => {
+  return (
```

Comment:
Do we need return statement here?

Reply:
I can remove it if you'd like to. I don't see any valuable difference.

Reply:
Ok, up to you